app/jobs/accession_complete_job.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+# Wait for an accession into SDR.
+class AccessionCompleteJob
+  include Sneakers::Worker
+  # This worker will connect to "preassembly.accession_complete" queue
+  # env is set to nil since by default the actual queue name would be
+  # "preassembly.accession_complete_development"
+  from_queue 'preassembly.accession_complete', env: nil
+
+  # This will receive both accession end-accession.completed and *.error messages.
```

Comment:
question due to my rabbit naivete: is the `from_queue 'preassembly.accession_complete'` enough to get this class to also receive `accession end-accession.completed`, due to some other rabbit config?  or is that wiring to come later?  something to do with the `queue.bind` in the `rabbit.rake`?  i've never been clear on what exactly topics and queues represent in rabbit (happy to go do some reading if this should be obvious).

Reply:
My head hasn't been it as recently as @justinlittman's, but, short answer: yes. Slightly longer answer is `from_queue` here tells sneakers which queue to connect to. The value matches what was set up on line 17 in the setup task:

https://github.com/sul-dlss/pre-assembly/pull/1247/files#diff-cfb3e45018fb1025f9ed2356abd6d3f9f58808e7ef269f9c1bf1380e9d9283a9R17

And then that queue was bound go the `sdr.workflow` topic using two routing keys (`end-accession.completed` and any errors) here:

https://github.com/sul-dlss/pre-assembly/pull/1247/files#diff-cfb3e45018fb1025f9ed2356abd6d3f9f58808e7ef269f9c1bf1380e9d9283a9R18-R19

The first five tutorials here cover this in greater detail:

https://www.rabbitmq.com/getstarted.html

Reply:
See the creation of the rabbit queue in `rabbit.rake`.

Reply:
thanks for the explanations and the doc link!